### ISSUE TITLE
Avocode not responding

### ISSUE THREAD

**Issue by exp-anoop** [avocode/avocode#2124]
---
* **Version**: 3.4.0-web.2
* **Web/Desktop**: Web
* **Platform**: linux -1
* **Browser**: chrome 68.0.3440.106
* **Avocode UID**: 174597
---
```
TypeError: Cannot read property 'destroyOutputAttributes' of undefined
    at e.finalize (https://cdn.app.avocode.com/dist/v-5e951c1/js/avocode-web.min.js:88:129574)
    at n.finalize (https://cdn.app.avocode.com/dist/v-5e951c1/js/avocode-web.min.js:88:144848)
    at https://cdn.app.avocode.com/dist/v-5e951c1/js/avocode-web.min.js:153:1153584
    at dt.__iterate (https://cdn.app.avocode.com/dist/v-5e951c1/js/avocode-web.min.js:1:26039)
    at dt.forEach (https://cdn.app.avocode.com/dist/v-5e951c1/js/avocode-web.min.js:1:50005)
    at https://cdn.app.avocode.com/dist/v-5e951c1/js/avocode-web.min.js:153:1153562
    at https://cdn.app.avocode.com/dist/v-5e951c1/js/avocode-web.min.js:1:17150
    at ze.iterate.We.iterate (https://cdn.app.avocode.com/dist/v-5e951c1/js/avocode-web.min.js:1:22901)
    at Ue.__iterate (https://cdn.app.avocode.com/dist/v-5e951c1/js/avocode-web.min.js:1:17119)
    at Ue.forEach (https://cdn.app.avocode.com/dist/v-5e951c1/js/avocode-web.min.js:1:50005)
    at http
```

**Comment by jakubzitny** [comment]
Hi @exp-anoop, thanks for reporting this. How many times did this happen to you, please?

Anyway, it's a bug and we'll release a new version today. It should fix this problem. We'll let you know when it's out so you can try it.

**Comment by jakubzitny** [comment]
We've released `3.4.1` few days ago, and this should be fixed in it. Please let us know if you see it again 😉
